app/views/deploys/_timeline_entry.html.erb

```diff
@@ -1,27 +0,0 @@
-<tr class="timeline-entry"
-  data-url="{{deploy.url}}" ng-click="jumpTo($event)"
-  ng-repeat="deploy in (deploys = (timelineDeploys.entries | projectUserFilter:search | userFilter:userType | stageFilter:stageType | statusFilter:deployStatus))">
```

Comment:
sadly no, the current deploys tab uses all the recent stuff, which in turn
loads all that JS.

On Mon, Apr 18, 2016 at 12:45 PM, Michael Grosser <notifications@github.com>
wrote:

> In app/views/deploys/_timeline_entry.html.erb
> <https://github.com/zendesk/samson/pull/834#discussion_r60121762>:
>
> > @@ -1,27 +0,0 @@
> > -<tr class="timeline-entry"
> > -  data-url="{{deploy.url}}" ng-click="jumpTo($event)"
> > -  ng-repeat="deploy in (deploys = (timelineDeploys.entries | projectUserFilter:search | userFilter:userType | stageFilter:stageType | statusFilter:deployStatus))">
>
> can get rid of a bunch of js now too like timelineDeploys / userFilter etc
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/834/files/77c775688b7d2d919e8adba6d1ef19c3fed791e8#r60121762>
>